src/main/java/types/DisplayType.java

```diff
@@ -39,6 +39,7 @@ public enum DisplayType {
      * @date 23 Apr 2017
      * @status added
      */
+    @Deprecated
```

Comment:
why annotating VNC as deprecated?

Reply:
Because VNC is a GraphicsType, and not a DisplayType. This was used before the introduction of GraphicsType. But as we have both now, we deprecate the DisplayType.VNC as you need to use GraphicsType.VNC.

Reply:
so let's deprecate DisplayType entirely then..

Reply:
Well not at this moment I think. On CentOS 8 you would like to have DisplayType.QXL with GraphicsType.VNC for example. Like you can set both in the UI at this moment.

And for headless you might want to return DisplayType.NONE also.

So I wouldn't deprecate DisplayType ..

Reply:
> And for headless you might want to return DisplayType.NONE also.

or keep the current design - you can achieve this by removing the current video devices

Reply:
> > And for headless you might want to return DisplayType.NONE also.
> 
> or keep the current design - you can achieve this by removing the current video devices

It's not a requirement as far as I see to have DisplayType.NONE indeed.
But does it hurt to have an additional way to make a VM headless by setting the DisplayType to NONE?